### PR TITLE
Adding short_title and adding countries to companies

### PR DESCRIPTION
**Pull request by persynmichael**
Removing the word - TutorCruncher

in the layout of features added short_title and then adjust the titels on every page

### CODE REVIEW

File: site/companies/brownstone-tutors.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ title: Brownstone Tutors
 telephone: 212-878-6698
 email: info@brownstonetutors.com
 website: http://www.brownstonetutors.com/
-address: Manhattan,275 Madison Ave., 4th Floor,new York, Ny 10016,354 Court Street, First Floor,brooklyn, Ny 11231,
-map_address: Manhattan,275 Madison Ave., 4th Floor,new York, Ny 10016,354 Court Street, First Floor,brooklyn, Ny 11231,
+address: Manhattan, 275 Madison Ave, 4th Floor
```
Comment (persynmichael) [inline site/companies/brownstone-tutors.md]: This

File: site/companies/clarendon-tutors.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ title: Clarendon Tutors
 telephone: 0207 206 2569
 email: info@clarendontutors.com
 website: http://clarendontutors.com
-address: Clarendon Tutors 15 Clarendon Street London Sw1v 2en
-map_address: Clarendon Tutors 15 Clarendon Street London Sw1v 2en
+address: Clarendon Tutors 15 Clarendon Street London SW1V 2EN
+map_address: Clarendon Tutors 15 Clarendon Street London SW1V 2EN
 description: Clarendon Tutors TutorCruncher Company Profile
-categories: company
+categories: companyS
```
Comment (persynmichael) [inline site/companies/clarendon-tutors.md]: This

File: site/companies/a1-tutors.md [hunk 3]
```diff
@@ -10,6 +10,8 @@ website: "http://www.privatetuition.co.uk/"
 address: "19 Annes Grove, Great Linford, Milton Keynes MK14 5DR"
 map_address: "19 Annes Grove, Great Linford, Milton Keynes MK14 5DR, United Kingdom"
 description: "A1 Tutors TutorCruncher Company Profile"
+country: United Kingdom
+city: Great Linford
```
Comment (tomhamiltonstubber) [inline site/companies/a1-tutors.md]: Milton Keynes

File: site/companies/apex-tutors-st-albans.md [hunk 4]
```diff
@@ -9,6 +9,8 @@ website: "http://www.apextutors.co.uk/"
 address: "42 Riverdene, Eadgware, HA8 9TD"
 map_address: "42 Riverdene, Eadgware, HA8 9TD, United Kingdom"
 description: "Apex Tutors TutorCruncher Company Profile"
+country: United Kingdom
+city: Edgware
```
Comment (tomhamiltonstubber) [inline site/companies/apex-tutors-st-albans.md]: London

File: site/companies/apex-tutors-st-albans.md [hunk 5]
```diff
@@ -9,6 +9,8 @@ website: "http://www.apextutors.co.uk/"
 address: "42 Riverdene, Eadgware, HA8 9TD"
 map_address: "42 Riverdene, Eadgware, HA8 9TD, United Kingdom"
```
Comment (tomhamiltonstubber) [inline site/companies/apex-tutors-st-albans.md]: Spelt wrong

File: site/companies/approved-tutors.md [hunk 6]
```diff
@@ -5,9 +5,11 @@ title: Approved Tutors
 telephone: 01670 734272
 email: 
 website: https://approvedtutors.co.uk
-address: 5 Kielder Ave, Cramlington, Northumberland Ne23 8jt
-map_address: 5 Kielder Ave, Cramlington, Northumberland Ne23 8jt
+address: 5 Kielder Ave, Cramlington, Northumberland NE23 8JT
+map_address: 5 Kielder Ave, Cramlington, Northumberland NE23 8JT
 description: Approved Tutors TutorCruncher Company Profile
+country: United Kingdom
+city: Cramlington
```
Comment (tomhamiltonstubber) [inline site/companies/approved-tutors.md]: Northumberland

File: site/companies/autism-partnership.md [hunk 7]
```diff
@@ -5,9 +5,11 @@ title: Autism Partnership
 telephone: 0113 250 9275
 email: info@autismpartnership.co.uk.
 website: http://www.autismpartnership.co.uk
-address: Victoria House Henshaw Lane, Yeadon, Leeds, Ls19 7rz
-map_address: Victoria House Henshaw Lane, Yeadon, Leeds, Ls19 7rz
+address: Henshaw Lane, Yeadon, Leeds, LS19 7RZ
+map_address: Henshaw Lane, Yeadon, Leeds, LS19 7RZ
 description: Autism Partnership TutorCruncher Company Profile
+country: United Kingdom
+city: Yeadon
```
Comment (tomhamiltonstubber) [inline site/companies/autism-partnership.md]: Leeds

File: site/companies/bonas-macfarlane.md [hunk 8]
```diff
@@ -6,9 +6,11 @@ founded: "1992"
 telephone: "+44 (0) 207 223 2794"
 email: "info@bonasmacfarlane.co.uk"
 website: "http://www.bonasmacfarlane.co.uk"
-address: "Bonas MacFarlane Education, Bridges Court, London"
-map_address: "Bonas MacFarlane Education, Bridges Court, London"
+address: "London"
```
Comment (tomhamiltonstubber) [inline site/companies/bonas-macfarlane.md]: Fix

File: site/companies/cambuslang-tutor.md [hunk 9]
```diff
@@ -5,9 +5,11 @@ title: Cambuslang Tutor
 telephone: 0141 571 2517
 email: scotland@kip-mcgrath.com
 website: http://www.kipmcgrathscotland.co.uk
-address: 13d Hunter Street, The Village East Kilbride, G74 4lz
-map_address: 13d Hunter Street, The Village East Kilbride, G74 4lz
+address: 13d Hunter Street, The Village East Kilbride, G74 4LZ
+map_address: 13d Hunter Street, The Village East Kilbride, G74 4LZ
 description: Cambuslang Tutor TutorCruncher Company Profile
+country: United Kingdom
+city: East Kilbride
```
Comment (tomhamiltonstubber) [inline site/companies/cambuslang-tutor.md]: Glasgow

File: site/companies/headway-tutors.md [hunk 10]
```diff
@@ -5,9 +5,11 @@ title: Headway Tutors
 telephone: 07590 035025
 email: info@headwaytutors.co.uk
 website: http://headwaytutors.co.uk
-address: Tw11 9jq Richmond Upon Thames, Surrey, United Kingdom
-map_address: Tw11 9jq Richmond Upon Thames, Surrey, United Kingdom
+address: Kingston Rd, Teddington TW11 9JQ
+map_address: Kingston Rd, Teddington TW11 9JQ
 description: Headway Tutors TutorCruncher Company Profile
+country: United Kingdom
+city: Teddington
```
Comment (tomhamiltonstubber) [inline site/companies/headway-tutors.md]: London

File: site/companies/select-tutors.md [hunk 11]
```diff
@@ -7,9 +7,11 @@ founded: "1994"
 telephone: "0500 179407"
 email: "info@select-tutors.co.uk"
 website: "http://www.select-tutors.co.uk/"
-address: "8 Cranmere Drive, SALE, Cheshire, M33 4LB"
-map_address: "8 Cranmere Drive, SALE, Cheshire, M33 4LB, United Kingdom"
+address: "8 Cranmere Drive, Sale, Cheshire, M33 4LB"
+map_address: "8 Cranmere Drive, Sale, Cheshire, M33 4LB, United Kingdom"
 description: "Select Tutors TutorCruncher Company Profile"
+country: United Kingdom
+city: Sale
```
Comment (tomhamiltonstubber) [inline site/companies/select-tutors.md]: Manchesta

File: site/companies/shine-tutoring-services.md [hunk 12]
```diff
@@ -5,9 +5,11 @@ title: Shine Tutoring Services
 telephone: 0780 4597 345
 email: 
 website: http://www.shinetutors.co.uk
-address: New Malden Surrey Kt3 5dn
-map_address: New Malden Surrey Kt3 5dn
+address: New Malden KT3
```
Comment (tomhamiltonstubber) [inline site/companies/shine-tutoring-services.md]: Thetford Rd, New Malden, Greater London KT3 5DT, UK

File: site/companies/shine-tutoring-services.md [hunk 13]
```diff
@@ -5,9 +5,11 @@ title: Shine Tutoring Services
...
+map_address: New Malden KT3
 description: Shine Tutoring Services TutorCruncher Company Profile
+country: United Kingdom
+city: New Malden
```
Comment (tomhamiltonstubber) [inline site/companies/shine-tutoring-services.md]: Ldon

File: site/companies/teach-me-group.md [hunk 14]
```diff
@@ -6,9 +6,11 @@ founded: 2012
 telephone: +44 (0) 800 689 0193
 email: info@shropshiretutor.co.uk
 website: http://www.shropshiretutor.co.uk
-address: The Shed, Dulwich Grange, Bratton, Telford, Shropshire, TF5 0EB
-map_address: "The Shed, Dulwich Grange, Bratton, Telford, Shropshire, TF5 0EB"
+address: Dulwich Grange, Bratton, Telford, Shropshire, TF5 0EB
+map_address: "Dulwich Grange, Bratton, Telford, Shropshire, TF5 0EB"
 description: "Teach Me Group TutorCruncher Company Profile"
+country: United Kingdom
+city: Bratton
```
Comment (tomhamiltonstubber) [inline site/companies/teach-me-group.md]: Telford

File: site/companies/tuition-connect.md [hunk 15]
```diff
@@ -5,9 +5,11 @@ title: Tuition Connect
 telephone:  0203 488 1750
 email: info@tuitionconnect.co.uk
 website: http://www.tuitionconnect.co.uk
-address: 
-map_address: 
+address: United Kingdom WS9 0LZ
```
Comment (tomhamiltonstubber) [inline site/companies/tuition-connect.md]: Mill Ln, Aldridge, Walsall, Staffordshire WS9 0LZ

File: site/companies/tutor-magic.md [hunk 16]
```diff
@@ -8,6 +8,8 @@ website: http://tutor-magic.co.uk
 address: Hampton In Arden, Solihull
 map_address: Hampton In Arden, Solihull
 description: Tutor Magic TutorCruncher Company Profile
+country: United Kingdom
+city: Hampton In Arden
```
Comment (tomhamiltonstubber) [inline site/companies/tutor-magic.md]: Solihull

File: site/companies/tutor-nanny-company.md [hunk 17]
```diff
@@ -7,9 +7,11 @@ founded: ""
 telephone: "44(0)203 004 7490"
 email: "caroline@tutornanny.co.uk"
 website: "http://www.tutornanny.co.uk/"
-address: "Riverbank House, 1 Putney Bridge Approach, Fulham, SW6 3JD"
-map_address: "Riverbank House, 1 Putney Bridge Approach, Fulham, SW6 3JD, United Kingdom"
+address: "1 Putney Bridge Approach, Fulham, SW6 3JD"
+map_address: "1 Putney Bridge Approach, Fulham, SW6 3JD, United Kingdom"
 description: "The Tutor Nanny Company TutorCruncher Company Profile"
+country: United Kingdom
+city: Fulham
```
Comment (tomhamiltonstubber) [inline site/companies/tutor-nanny-company.md]: London

File: site/companies/woodlands-tuition.md [hunk 18]
```diff
@@ -5,9 +5,11 @@ title: Woodlands Tuition
 telephone: 07535 432233
 email: woodlandstuition@gmail.com
 website: http://www.woodlandstuition.co.uk
-address: Woodlands Tuition 95 Ashmead Road Bedford Bedfordshire Mk41 7fd
-map_address: Woodlands Tuition 95 Ashmead Road Bedford Bedfordshire Mk41 7fd
+address: 95 Ashmead Road Bedford Bedfordshire MK41 7FD
+map_address: 95 Ashmead Road Bedford Bedfordshire MK41 7FD
 description: Woodlands Tuition TutorCruncher Company Profile
+country: United Kingdom
+city: Bedford
```
Comment (tomhamiltonstubber) [inline site/companies/woodlands-tuition.md]: Milton Keynes

**Pull request merged**